lightning/src/blinded_path/message.rs

```diff
@@ -74,16 +68,106 @@ impl Writeable for ForwardTlvs {
 	}
 }
 
+/// Similar to [`ForwardTlvs`], but these TLVs are for the final node.
+pub(crate) struct ReceiveTlvs {
+	pub tlvs: Option<RecipientData>
+}
+
+impl ReceiveTlvs {
+	pub fn new(tlvs: Option<RecipientData>) -> Self {
+		ReceiveTlvs { tlvs }
+	}
+}
+
+/// Represents additional data appended along with the sent reply path.
```

Comment:
This is not restricted to reply paths.

s/appended .../included by the recipient in a [`BlindedPath`].

Reply:
Done! [8e4e1aa](https://github.com/lightningdevkit/rust-lightning/commit/8e4e1aabadb5fda9f4d0ec2c845263c94c273643)
Thanks!